3.assess_generalizability/scripts/generalizabilty_eval.py

```diff
@@ -0,0 +1,324 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Evaluate generalizability of the model across holdout plate(s)
+
+# ## Import libraries
+
+# In[1]:
+
+
+import pandas as pd
+import pathlib
+from joblib import load
+import numpy as np
+from sklearn.metrics import (
+    accuracy_score,
+    precision_recall_curve,
+)
+from typing import Tuple
+import seaborn as sns
+import matplotlib.pyplot as plt
+
+
+# ## Define helper functions
+
+# In[2]:
+
+
+def get_X_y_data(
```

Comment:
Just checking, don't you have this function elsewhere? If not, then should we? If not, then should we name it differently to be more specific about what it does?

Reply:
Ahh I had it copied from another repo (CFReT) which is private and I also made my own edits. I don't think we need it elsewhere, but I will make it more specific. We can also chat more about this later.

Reply:
Wait nevermind we don't even use it, I will just remove it!